rd/timestep.cpp

```diff
@@ -79,7 +83,7 @@ void drift_update(int TBIN_CURRENT, int N_TRIANG, double T, double DT, std::vect
         }
 }
 
-void reset_tbins(int T, int DT, int N_TRIANG, int N_POINTS, double &NEXT_DT, std::vector<TRIANGLE> &RAND_MESH, std::vector<VERTEX> &RAND_POINTS){
+void reset_tbins(double T, double DT, int N_TRIANG, int N_POINTS, double &NEXT_DT, std::vector<TRIANGLE> &RAND_MESH, std::vector<VERTEX> &RAND_POINTS){
```

Comment:
T and DT are double, though converting them to int implicitly won't change the simulation results.